science/nds2-client/Portfile

```
@@ -16,24 +16,25 @@ long_description \
   Client tool for accessing streamed LIGO data using the Network \
   Data Server version 2.
 
-homepage          https://wiki.ligo.org/DASWG/NDSClient
+homepage          https://wiki.ligo.org/Computing/NDSClient
 master_sites      https://software.igwn.org/lscsoft/source/
+use_bzip2         yes
 
-checksums         rmd160  9653ad19cc2af2d194c8331e5cac99c6beb95119 \
-                  sha256  485ce5cc1650f384897e5ddcd259ad2f4f6b63e309372990b997d86b54cf529e \
-                  size    535845
+checksums         rmd160  d838d9602fc7e5258fae67b47fddaca99b58ea9e \
+                  sha256  a6d83cc0f96f63ca7e045f3923dc178ad48a4869d61d5e2086d184c7965e5e15 \
+                  size    583936
 
-depends_build-append \
+depends_build \
+                  port:cmake \
                   port:doxygen \
-                  path:bin/dot:graphviz \
-                  port:pkgconfig
+                  port:graphviz \
```

You should revert this change. It was correct to depend on graphviz using the `path` syntax because there are multiple ports that provide the graphviz software -- graphviz and graphviz-devel -- and this syntax allows either of them to satisfy the dependency.
```suggestion
                  path:bin/dot:graphviz \
```

---

I have added this to my latest commit